src/components/KCard/KCard.vue

```diff
@@ -97,7 +97,7 @@ const showCardHeader = computed((): boolean => {
     display: flex;
     flex-direction: column;
     height: 100%;
-    width: 100%;
+    width: 100%
```

Comment:
oops, missing the semicolon
```suggestion
    width: 100%;
```

Reply:
Fixed